spec/factories/in_progress_forms.rb

```diff
@@ -237,5 +237,156 @@
+          ],
+          "view:reportDependents": true,
+          "spouseFullName": {
+            "first": 'Jane',
+            "middle": 'Pam',
+            "last": 'Doe',
+            "suffix": 'II'
+          },
+          "spouseSocialSecurityNumber": '232422344',
+          "spouseDateOfBirth": '1980-01-02',
+          "dateOfMarriage": '2004-01-02',
+          "cohabitedLastYear": false,
+          "provideSupportLastYear": true,
+          "sameAddress": false,
+          "view:spouseContactInformation": {
+            "spouseAddress": {
+              "street": '123 maple st',
+              "street2": 'Apt 1',
+              "street3": 'Floor 2',
+              "city": 'Florence',
+              "country": 'USA',
+              "state": 'MA',
+              "postalCode": '01060'
+            },
+            "spousePhone": '3424445555'
+          },
+          "discloseFinancialInformation": true,
+          "vaCompensationType": 'highDisability',
+          "purpleHeartRecipient": true,
+          "isFormerPow": true,
+          "postNov111998Combat": true,
+          "disabledInLineOfDuty": true,
+          "swAsiaCombat": true,
+          "vietnamService": true,
+          "exposedToRadiation": true,
+          "radiumTreatments": true,
+          "campLejeune": true,
+          "lastServiceBranch": 'air force',
+          "lastEntryDate": '2000-01-02',
+          "lastDischargeDate": '2005-02-01',
+          "dischargeType": 'general',
+          "email": 'test@test.com',
+          "view:emailConfirmation": 'test@test.com',
+          "homePhone": '5555555555',
+          "mobilePhone": '4444444444',
+          "veteranAddress": {
+            "street": '123 aspen st',
+            "street2": 'Apt 4',
+            "street3": 'Room 6',
+            "city": 'Hadley',
+            "country": 'USA',
+            "state": 'MA',
+            "postalCode": '01070'
+          },
+          "gender": 'M',
+          "maritalStatus": 'Married',
+          "view:demographicCategories": {
+            "isSpanishHispanicLatino": true,
+            "isAmericanIndianOrAlaskanNative": true,
+            "isBlackOrAfricanAmerican": true,
+            "isNativeHawaiianOrOtherPacificIslander": true,
+            "isAsian": true,
+            "isWhite": true
+          },
+          "veteranDateOfBirth": '1980-03-02',
+          "veteranSocialSecurityNumber": '324234444',
+          "view:placeOfBirth": {
+            "cityOfBirth": 'Boston',
+            "stateOfBirth": 'MA'
+          },
+          "veteranFullName": {
+            "first": 'Jim',
+            "middle": 'Bob',
+            "last": 'Doe',
+            "suffix": 'Jr.'
+          },
+          "mothersMaidenName": 'Smith',
+          "privacyAgreementAccepted": true,
+          "testUploadFile": {
+            "filePath": '/src/platform/testing/',
+            "fileName": 'example-upload.png',
+            "fileTypeSelection": '7'
+          }
+        }.to_json
+      end
+    end
```

Comment:
We can address this in a follow-up PR, but IMO this file is getting a bit long. We should probably break each form type out into its own file. Maybe `factories/in_progress_forms/hca_form.rb`.